certified-connectors/VirusTotal/readme.md

```diff
@@ -0,0 +1,20 @@
+ ### NOTE
+> This is a *sample* connector.  The connector is provided here with the intent to illustrate certain features and functionality around connectors.
```

Comment:
This is a certified connector, not a sample one. So this whole note should be removed.